submissions/yhorodechna/frogger-oop/app.js

```diff
@@ -0,0 +1,156 @@
+const SCORE = 0;
+const CANVAS = {
+    width: 505,
+    height: 605,
+    cellHeight: 83,
+    cellWidth: 101,
+};
+/* The function RESET_ENEMIES_SPEED add for each enemy random speed. This function is used in class Enemy 
+to reset the speed of each enemy when a player loses */
+const RESET_ENEMIES_SPEED = function () {
+    for (let i = 0; i < allEnemies.length; i++) {
+        allEnemies[i].speed = Math.floor(Math.random() * (100 - 40)) + 40;
+    }
+};
```

Comment:
(1) When comment explains **what** is done instead of **why** that means that the code is most likely not expressive enough.

(2) An enemy position reset per se should be the enemy's behavior (read "class method"), as class properties handling should be encapsulated in class. Of course, bulk processing is OK to have beyond individual class definition.

Reply:
Created new method in Enemy class